app/models/division.rb

```diff
@@ -50,8 +50,7 @@ def self.root_internal_name
 
   def self.root
     # create on demand if not present for convenience of blank db's and test cases
-    # @@root ||=
```

Comment:
I've actually been meaning to mention this, but [closure_tree already supports getting the root](https://github.com/mceachen/closure_tree#class-methods), and I'm not sure whether I understand the impetus behind switching from `find_or_create_by` to `find_by || create`. If the only issue is that you want to create it with more attributes than you want to find it by, you can just use `Division.find_or_create_by(internal_name: 'root').update_attributes(name: 'Root Division')`